src/transformers/modeling_funnel.py

```diff
@@ -598,8 +598,8 @@ def __init__(self, config, block_index):
         self.attention = FunnelRelMultiheadAttention(config, block_index)
         self.ffn = FunnelPositionwiseFFN(config)
 
-    def forward(self, q, k, v, attention_inputs, output_attentions=False):
-        attn = self.attention(q, k, v, attention_inputs, output_attentions=output_attentions)
+    def forward(self, query, key, value, attention_inputs, output_attentions=False):
```

Comment:
@patrickvonplaten This is just for you :)

Reply:
haha :-)